codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameReader.java

```diff
@@ -111,8 +112,8 @@ public Http2FrameSizePolicy frameSizePolicy() {
     @Override
     public void maxFrameSize(int max) throws Http2Exception {
         if (!isMaxFrameSizeValid(max)) {
-            throw streamError(streamId, FRAME_SIZE_ERROR,
-                    "Invalid MAX_FRAME_SIZE specified in sent settings: %d", max);
+            // SETTINGS frames affect the entire connection state and thus errors must be connection errors.
```

Comment:
Would be helpful to add a link to the corresponding RFC section: https://datatracker.ietf.org/doc/html/rfc9113#section-4.2
(same comment for other places)

Reply:
Done.